client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -52,24 +52,14 @@ const AboutUkraineContainer = styled((props) => <Box {...props} />)(({ theme })
     },
 }));
 
-const fadeIn = keyframes`
-0% {
-    opacity: 0;
-  }
-  100% {
-    opacity: 1;
-  }
-`;
-
 const UkraineBackground = styled((props) => <Box {...props} />)(({ theme }) => ({
     height: 749,
-    backgroundImage: `url(${"../../assets/imgAboutUkraineSection/ukraine.jpeg"})`,
+    backgroundImage: `url(${"./assets/images/aboutUkraineSection/ukraine.jpeg"})`,
```

Comment:
background image not working. please fix

![Uploading image.png…]()

Reply:
Done